docs/samples/custom/timeseries-holtwinter/model.py

```diff
@@ -0,0 +1,52 @@
+import kfserving
+from typing import List, Dict
+import base64
+import io
+import numpy as np
+import joblib
+import pickle
+
+
+class KFServingSampleModel(kfserving.KFModel):
+    def __init__(self, name: str):
+        super().__init__(name)
+        self.name = name
+        self.ready = False
+
+    def load(self):
+       
+        #self.model =  joblib.load('model.sav')
+        self.ready = True
+
+    def predict(self, request: Dict) -> Dict:
+
+
+        
+        inputs = request["instances"]
+
+        
+        weeks = int(inputs[0]["image"]["weeks"])
+        print(weeks)
+
+        #Loading the joblib files 
+        self.holt =  joblib.load('model_holt.sav')
+        
+
+
+        #creating return dictionary 
+        dict1={"sales":"application"}
```

Comment:
Ideally we'd like to make predict generic and application specific processing can be moved to `preprocess` or `postprocess` handler.

Reply:
@yuzisun  Let me change it